index.html

```diff
@@ -388,7 +388,6 @@ <h2>Key Discovery</h2>
   <p class="issue" data-number="31"></p>
   <p class="issue" data-number="30"></p>
   <p class="issue" data-number="15"></p>
-  <p class="issue" data-number="13"></p>
```

Comment:
<img width="1681" alt="Screen Shot 2023-07-07 at 5 44 32 PM" src="https://github.com/w3c/vc-jwt/assets/8295856/b534f9ca-59ab-4cc9-9ace-23a515cc99c7">

Reply:
the issue marker plugin imports `input` tags... which then break validation... also... seems dangerous to be importing `input` tags from an issue tracker...

Reply:
> the issue marker plugin imports `input` tags... which then break validation... also... seems dangerous to be importing `input` tags from an issue tracker...

It seems like this should be raised as an issue on the issue marker plugin, rather than a comment within a PR (which will persist somewhere, but may be generally invisible, after the PR is merged)?

Reply:
I'm not trying to contribute to echidna or respec.

I'm explaining why the commits for this PR were failing for reasons unrelated to the changes in the PR.

You are welcome to cross link to this, if find the right repo and want to help them fix this bug.

Reply:
> I'm not trying to contribute to echidna or respec.

Go, team spirit!

> I'm explaining why the commits for this PR were failing for reasons unrelated to the changes in the PR.

Usually, that would lead me to report an issue on the component that was causing me a headache, whether or not I could provide further input toward resolving that issue. Often, others chime in with "I hit that, too" or "here's how I worked around that" or "I've created a PR with a potential fix".

> You are welcome to cross link to this, if find the right repo and want to help them fix this bug.

@msporny @iherman — Does either of you know the repo for the issue marker plugin?

I do not know enough about what was going wrong to usefully report the issue there, but perhaps if the repo becomes known, @OR13 will find a few minutes to log the issue.

Reply:
@TallTed I pay membership fees to to W3C... I also volunteer a lot of time... but I also agree with your team spirit comment... I think... 

```
I'm worse at what I do best
And for this gift, I feel blessed
Our little group has always been
And always will until the end
```

- Nirvana